src/web/lib/components/AppLoadingIndicator/index.js

```diff
@@ -1,15 +1,13 @@
 import React from "react";
 import "./index.scss";
-import { loaderQuotes } from "../../../../lib/constants";
 
-export const AppLoadingIndicator = () => {
-  const i = Math.floor(Math.random() * loaderQuotes.length);
```

Comment:
Turns out this was selecting a new quote every time the app renders - which is every time the state changes. And the state changes several times during loading, which is why we have a spinner :)